docs/topics/downloader-middleware.rst

```diff
@@ -681,7 +681,9 @@ HttpProxyMiddleware
    * ``no_proxy``
 
    You can also set the meta key ``proxy`` per-request, to a value like
-   ``http://some_proxy_server:port``.
+   ``http://some_proxy_server:port``. Keep in mind this value will take
+   precedence over ``http_proxy``/``https_proxy`` enviroment variables, and
```

Comment:
Typo here: `environment`.

Also, might as well mention that the proxy value can have crendentials in it.